acceptance/acceptance_test.go

```diff
@@ -78,6 +83,10 @@ func writeSampleGCPolicyInput(ctx context.Context, sampleName string) (context.C
 		content = sampleGCPolicyInput
 	case "clamav-task":
 		content = sampleClamAVTask
+	case "v02-sample-attestation":
```

Comment:
With `embed.FS`, this becomes something like:

```
	sampleData, err := sample.ReadFile(filepath.Join("sample", sampleName + ".json")
	if err != nil {
		return ctx, fmt.Errorf("reading sample %s: %w", sampleName, err)
	}

	return ctx, os.WriteFile(ts.inputFileName, sampleData, 0400)
```

Reply:
I like this for a future PR.